logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultAnyAssertions.kt

```diff
@@ -52,4 +53,13 @@ class DefaultAnyAssertions : AnyAssertions {
         container.changeSubject.reportBuilder()
             .downCastTo(subType)
             .build()
+
+    override fun <T> isNotIn(container: AssertionContainer<T>, expected: List<T>): Assertion {
```

Comment:
My fault, `List` is actually not required, let's rewrite to `Iterable`
```suggestion
    override fun <T> isNotIn(container: AssertionContainer<T>, expected: Iterable<T>): Assertion {
```